src/features/home/components/HomeBodyPlaceholder.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import { HomeBodyPlaceholder } from './HomeBodyPlaceholder'
+
+export default {
+  title: 'ui/placeholders',
+  component: HomeBodyPlaceholder,
+} as ComponentMeta<typeof HomeBodyPlaceholder>
+
+export const HomeBody: ComponentStory<typeof HomeBodyPlaceholder> = () => <HomeBodyPlaceholder />
```

Comment:
je propose de supprimer `storyName`

```suggestion
export const HomeBodyPlaceholder: ComponentStory<typeof HomeBodyPlaceholder> = () => <HomeBodyPlaceholder />
```

Reply:
Il y a un conflit avec l'import si on utilise le même nom. Du coup je laisse comme ça ?

Reply:
sinon `Default` ?

Reply:
Je dois quand même laisser le storyName sinon on a Default :

<img width="225" alt="Capture d’écran 2022-03-30 à 14 36 32" src="https://user-images.githubusercontent.com/78556024/160837311-3ead4fb9-c242-4d56-a71a-1f292ebf67b6.png">